### DISCUSSION TITLE
Neshta.Virus.Filelnfector.DDS detection by Malwarebytes

### DISCUSSION THREAD

**Discussion by RealJamieRI** [valinet/ExplorerPatcher#3344] [Q&A]
My Malwarebytes Premium just announced it detected Neshta.Virus.Filelnfector.DDS 
![image](https://github.com/valinet/ExplorerPatcher/assets/48834037/8b916857-4c20-4ea7-a27d-fbaca80756a5)

HTTPS//GITHUB.COMALINET/EXPLORERPATCHER/RELEASES/LATEST/DOWNLOAD/EP_SETUP.EXE FROM EXPLORERPATCHER FOR C:\USERS\JAMIE\APPDATA\ROAMING\EXPLORERPATCHER\UPDATE

I went to the folder C:\Users\jamie\AppData\Roaming\ExplorerPatcher and found this file and checked it's properties, but it had none. Then the file disappeared from the folder. 

Now Microsoft Defender says something different: 

Detected: HackTool:Win64/ExplorerPatcher!MTB
Status: Quarantined
Quarantined files are in a restricted area where they can't harm your device. They
will be removed automatically.
Date: 6/1/2024 5:06 PM
Details: This program has potentially unwanted behavior.
Affected items: 
file: C:\Users\jamie\AppData\Roaming\ExplorerPatcher\Update for
ExplorerPatcher from
https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_set
up.exe
file: \\localhost\C$\Users\jamie\AppData\Roaming\ExplorerPatcher\Update
for ExplorerPatcher from
https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_set
up.exe

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.

MS just doesn't like EP and has added it to windows defender, even though it is not a virus and not a hack tool.

Set windows defender to exclude the following:
1. C:\Program Files\ExplorerPatcher
2. %APPDATA%\ExplorerPatcher

Future updates to EP won't be flagged then.

You can also set windows defender to exclude the directory that you manually download EP to so you can install it without windows defender blocking it.

**Reply by RealJamieRI** [reply]
Not wholly a duplicate. 
#3228 only discusses Microsoft Defender. 
What about Neshta.Virus.Filelnfector.DDS detection by Malwarebytes, the title of my discussion? 
[https://www.malwarebytes.com/blog/detections/neshta-virus-fileinfector-dds](https://www.malwarebytes.com/blog/detections/neshta-virus-fileinfector-dds)

**Comment by pyrates999** [comment]
That's a false positive.
